addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -707,6 +732,35 @@ import { sprintf } from "@web/core/utils/strings";
                 inputEl.disabled = !haveToBeVisible;
             }
         },
+        /**
+         * Creates a block containing the file name and a cross to delete it.
+         *
+         * @private
+         * @param {Object} fileDetails the details of the file being uploaded
+         * @param {HTMLElement} filesZoneEl the zone where the file blocks are
+         *      displayed
+         */
+        _createFileBlock(fileDetails, filesZoneEl) {
+            const fileBlockEl = qweb.render('website.file_block', {fileName: fileDetails.name});
+            filesZoneEl.insertAdjacentHTML('beforeend', fileBlockEl);
+            filesZoneEl.lastElementChild.fileDetails = fileDetails;
+        },
+        /**
+         * Creates the file upload button (= a button to replace the file input,
+         * in order to modify its text content more easily).
+         *
+         * @private
+         * @param {HTMLElement} inputEl the file input
+         */
+        _createAddFilesButton(inputEl) {
+            const addFilesButtonEl = document.createElement('INPUT');
+            addFilesButtonEl.classList.add('o_add_files_button', 'form-control');
+            addFilesButtonEl.type = 'button';
+            addFilesButtonEl.title = inputEl.title;
+            addFilesButtonEl.value = inputEl.getAttribute('multiple') != null ? _t("Add Files") : _t("Replace File");
```

Comment:
You can get rid of `!= null`, if inputEl hasn't the attribute it will be falsy, also line length

Reply:
Now that I removed the multiple = 1, when we get the `multiple` attribute it is an empty string, which is also falsy 
So I need to compare it to `null` 

I will need to add a comparison to null in `_onFileChange(ev)` too when I check if there is the multiple attribute

Reply:
You can use [Element.hasAttribute()](https://developer.mozilla.org/en-US/docs/Web/API/Element/hasAttribute)